VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/tests/judge-waiting-room.component.spec.ts

```diff
@@ -367,6 +367,13 @@ describe('JudgeWaitingRoomComponent when conference exists', () => {
             await component.retrieveAudioStreamInfo(globalConference.id);
             expect(component.conferenceRecordingInSessionForSeconds).toBe(0);
         });
+
+        it('should switch the continueWithNoRecording flag to false when conference is not in session', async () => {
```

Comment:
Could you also test that it doesn't change when it is in session.

Reply:
done